prog/weaveexec/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM alpine
+#FROM alpine
+FROM jeanblanchard/alpine-glibc
```

Comment:
but the golang CNI API itself doesn't have such a dependency, and that is all we are invoking here.